dev-lang/icon/icon-9.5.1-r1.ebuild

```diff
@@ -0,0 +1,119 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic toolchain-funcs
+
+MY_PV=${PV//./}
+SRC_URI="http://www.cs.arizona.edu/icon/ftp/packages/unix/icon-v${MY_PV}src.tgz"
+HOMEPAGE="http://www.cs.arizona.edu/icon/"
+DESCRIPTION="very high level language"
+
+LICENSE="public-domain HPND"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
+IUSE="X iplsrc"
+
+S="${WORKDIR}/icon-v${MY_PV}src"
+
+DEPEND="
+	X? (
+		x11-proto/xextproto
+		x11-proto/xproto
+		x11-libs/libX11
+		x11-libs/libXpm
+		x11-libs/libXt )
+	|| ( sys-devel/gcc sys-devel/gcc-apple )"
```

Comment:
depending on compilers in `DEPEND` is pointless, remove

Reply:
Not sure if clang has been tested upstream, but I can remove the depend.